examples/motion_viewer.py

```diff
@@ -63,7 +64,35 @@ def __init__(
         self.select_box_obj_id: int = -1
 
         # shortest path attributes
-        self.path_traj_obj_id = -1
+        self.spline_path_traj_obj_id = -1
+
+        self.navmesh_config_and_recompute()
+
+    def debug_draw(self):
+        """
+        Additional draw commands to be called during draw_event.
+        """
+
+        def draw_frame():
```

Comment:
This function is duplicated in fairmotion_interface.py. Maybe break it out into a common utility location to avoid duplication?

Reply:
Even though they both have the same code, they are different in that the motion_viewer is viewer rate called and the fairmotion one is called only during simulation. They are different but the reason I left the code to be the generic origin print is because I meant for it to be demo of what devs can do with the debug methods. It's like a head start for coding up debug lines so I though the most generic code to leave would be that. But if I move the origin code to a utility then it defeats the purpose of the head start code because devs would have to search for it in the utilities class and copy paste from there to begin adding debug draws.